src/Microsoft.Health.Client/Authentication/OAuth2ClientCertificateCredentialProvider.cs

```diff
@@ -76,6 +76,7 @@ protected override async Task<string> BearerTokenFunction(CancellationToken canc
 
     internal static string GenerateClientAssertion(string clientId, X509Certificate2 x509Certificate2, Uri tokenUri)
     {
+        // TODO: Migrate to using JsonWebToken
```

Comment:
Is this in this PR or for later?

Reply:
This is for later and anyone that looks at it. The `JsonWebToken` type seems to be newer and more performant